docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: '3.4'
+
+services:
+  http:
+    build: .
+    tty: true
```

Comment:
Unnecessary unless we require interactive-ness. Doesn't look like we need an interactive tty in this app.